src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -30,8 +30,8 @@ internal sealed partial class LocalRewriter : BoundTreeRewriterWithStackGuard
         private bool _sawAwait;
         private bool _sawAwaitInExceptionHandler;
         private bool _needsSpilling;
+        private Instrumenter _instrumenter;
```

Comment:
>private Instrumenter _instrumenter; [](start = 8, length = 35)

It is probably better to not move the line because we do not have a requirement to group readonly fields together.